files/default/pcluster_connect.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+
+# This script takes as input the shared folder. So for example you should called it like
+# ./pcluster_connect.sh "/shared"
+
+# Return the sessionid, the port and the tokenid
+
+create_new_session() {
+    dcv_path=$1
+    shared_folder_path=$2
+
+    dcv_file="${dcv_path}/dcv_session"
+    sessionid=$(shuf -zer -n20  {A..Z} {a..z} {0..9})
+    echo ${sessionid} > ${dcv_file}
+    dcv create-session --type virtual --storage-root ${shared_folder_path} ${sessionid}
+
+    echo ${sessionid}
+}
+
+main() {
+    shared_folder_path=$1
+    user=$(whoami)
+    os=$(cat /tmp/dna.json | jq -r .cfncluster.cfn_base_os)
+
+    if [[ ${os} != "centos7" ]]; then
+        >&2 echo "Non supported os"
+        exit 1
+    fi
+
+    if (( $(systemctl status dcvserver | grep inactive | wc -l) == 1 )); then
+        >&2 echo "DCV is not active on the given instance"
+        exit 1
+    fi
+
+    dcv_path="${HOME}/.parallelcluster_dcv"
```

Comment:
we should use a subfolder of the existing parallelcluster folder, so:
${HOME}/.parallelcluster/dcv

Reply:
a folder does not exists in the homedirectory for parallel cluster, but we can do it for sure